src/nxt_unit.c

```diff
@@ -3506,6 +3506,10 @@ nxt_unit_mmap_get(nxt_unit_ctx_t *ctx, nxt_unit_port_t *port,
 
     outgoing_size = lib->outgoing.size;
 
+    if (nxt_slow_path(outgoing_size == 0)) {
+        goto skip;
+    }
+
     mm_end = lib->outgoing.elts + outgoing_size;
```

Comment:
I assume this is because _lib->outgoing.elts_ can be NULL? Then I think it's better/clearer to check for that